### ISSUE TITLE
[crash] java.lang.NullPointerException: Initializing game

### ISSUE THREAD

**Issue by Yourname942** [CyclopsMC/IntegratedTerminals#56]
#### Issue type:

- :bug: Bug

____

#### Short description:
It seems that integrated terminals is breaking, which kills the block registering, so things later crash

#### Steps to reproduce the problem:
1. Select Play in Twitch App
2. Select Play in Minecraft Launcher
3. -- Crash here --

#### Expected behaviour:

I expect there to be no crashes when all I did was click play in the Minecraft launcher

____

#### Versions:
- Modpack [Here](https://drive.google.com/file/d/1h4XZCODiPQZrCfHCMQFIFCfFio1A4jNL/view?usp=sharing)
- Integrated Crafting: 1.0.10
- Integrated Dynamics: 1.1.16
- **Integrated Terminals:** 1.0.14 (I believe this mod to be the cause)
- Integrated Tunnels: 1.6.13
- Minecraft: 1.15.2
- Forge: 31.2.36

#### Log file:
latest log: [Here](https://gist.github.com/Yourname942/f290b90875d6546c01fdedad92cb1970)
Crash log: [Here](https://gist.github.com/Yourname942/a1549b6477f5023aa4a79c131b10d64c)

**Comment by rubensworks** [comment]
Thanks for reporting!

**Comment by HipHopHuman** [comment]
**tl;dr**: If it's hard to reproduce this issue, try installing any mod that depends on MixinBootstrap *after* installing Integrated Terminals.

I've been experiencing the same issue and I think it may be a tough one to reproduce.

I was making a modpack with very few mods (like 80 or so) and experienced the crash.

I installed MoreCrashInfo to determine that it was in fact Terminals causing the crash, but it seemd to work sometimes, sometimes not.

So, I made a fresh pack with the exact same mods - no crash. Worked just fine. Over 3 days of test/play time.

Then I installed Botania Tweaker. It started crashing. I uninstalled Botania Tweaker and it continued crashing. So, I deleted Integrated Terminals, loaded my world and the game worked again. I re-installed Terminals and there was no crash anymore. I noticed that Botania Tweaker has a dependency on MixinBootstrap. 

I decided to try another mod that depends on MixinBootstrap in a fresh pack with just the integrated suite. First, I installed the Integrated suite of mods, loaded the game, loaded a world, everything worked. Then I installed a mod that has a dependency on MixinBootstrap and the crash happened. I tried again to make a fresh pack and do it the other way this time - install MixinBootstrap first, then the Integrated suite. The game worked perfectly.

[Crash + Log file](https://gist.github.com/HipHopHuman/a1fcd69f0c58bafd5dafe99c9f50c8ab)

**Comment by rubensworks** [comment]
Thanks for the additional information @HipHopHuman, this will definitely make debugging easier.
